ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -510,6 +510,46 @@ def test_torch_cartesian_prod(
     )
 
 
+# cdist
+@handle_frontend_test(
+    fn_tree="torch.cdist",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=[
```

Comment:
why only limit to `float32` and `float64`? I think you should use `avalaible_dtypes=valid` here